constraint_utils/constraint_basics.py

```diff
@@ -635,9 +709,23 @@ def collapse_lof_ht(lof_ht: hl.Table, keys: Tuple[str], calculate_pop_pLI: bool
 def annotate_constraint_groupings(ht: Union[hl.Table, hl.MatrixTable],
                                   custom_model: str = None) -> Tuple[Union[hl.Table, hl.MatrixTable], List[str]]:
     """
-    HT must be exploded against whatever axis
-
-    Need to add `'coverage': ht.exome_coverage` here (which will get corrected out later)
+    Add constraint annotations.
+    
+    Function adds the following annotations:
+        - annotation - could be 'most_severe_consequence' of either 'worst_csq_by_gene' or 'transcript_consequences', or 'csq' of 'tx_annotation'
+        - modifier - classic lof annotation, LOFTEE annotation, or PolyPhen annotation
+        - gene
+        - coverage
+        - expressed (optional)
+        - transcript
+        - canonical
+    
+    ..note::
+        HT must be exploded against whatever axis.
```

Comment:
My understanding is that this corresponds to L514, L521, L535, so I kept this as a note. Correct me if I'm wrong